File: 5/smallest_multiple.lua
function Smallest_Multiple(x)
    local n = 1
    while true do

        for i = 1,x do
            if n % i ~= 0 then
                break
            elseif i == x then
                return n
            end
        end

        n = n + 1
        -- print (i, divisible)
        -- if divisible then print (i, divisible) return end

    end
end

print(Smallest_Multiple(20))

File: 5/smallest_multiple.jl

function smallest_multiple(x)
    n = 1
    while true
        #println(n)
        for i = 1:x
            #println("i = ", i)
            if n % i != 0
                break
            elseif i == x
                return n
            end
        end
        n += 1
    end
end

println(smallest_multiple(20))

